packages/@aws-cdk/aws-lambda/test/test.lambda.ts

```diff
@@ -186,19 +186,6 @@ export = {
       test.done();
     },
 
-    'fails if the principal is not a service or account principals'(test: Test) {
```

Comment:
Leave this test, but use a different type of principal (even mock one that doesn't have any of the options)

Reply:
I used OrganizationPrincipal